addons/stock/tests/test_stock_lot.py

```diff
@@ -72,3 +72,16 @@ def test_single_location(self):
         self.StockQuantObj.invalidate_model()
         self.StockQuantObj._unlink_zero_quants()
         self.assertEqual(self.lot_p_a.location_id, self.locationC)
+
+    def test_import_lots(self):
+        vals = self.MoveObj.action_generate_lot_line_vals({
+            'default_tracking': 'lot',
+            'default_product_id': self.productA.id,
+            'default_location_dest_id': self.locationC.id,
+        }, "import", "", 0, "aze;2\nqsd;4")
```

Comment:
Put also a line without any quantity and check `1` is taken to create the move line